api/libproject/devstart.rb

```diff
@@ -241,11 +239,34 @@ def run_with_creds(args, command, proc)
 def do_run_with_creds(project, account, creds_file, proc)
   if creds_file == nil
     service_account_creds_file = Tempfile.new("#{project}-creds.json")
-    get_service_account_creds_file(project, account, service_account_creds_file)
-    begin
-      proc.call(project, account, service_account_creds_file.path)
-    ensure
-      delete_service_accounts_creds(project, account, service_account_creds_file)
+    if project == "all-of-us-workbench-test"
+      creds_filename = "sa-key.json"
+      # For test, use a locally stored key file copied from GCS (which we leave hanging
+      # around.)
+      if !File.file?(creds_filename)
+        # Create a temporary creds file for accessing GCS.
+        get_service_account_creds_file(project, account, service_account_creds_file)
+        begin
+          activate_service_account(service_account_creds_file.path)
```

Comment:
We could even do this step (fetching a file from GCS) with user account creds instead of generate-and-delete a SA key. It'd run slightly faster, but this is a rare step and we already have nice features in place to support the SA-based operation.

Reply:
I don't know how to do that, using gsutil anyway... it seems to want you to activate a service account before you use it. There is no --account option or anything.

Is there a better way of doing this?

Reply:
Going to merge this for now, we can think of a better solution later hopefully. :)